Tests/test_api/base.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from flask import Flask
+from flask_testing import LiveServerTestCase
+
+from kalliope.core import LifoManager
+from kalliope.core.ConfigurationManager import BrainLoader
+from kalliope.core.ConfigurationManager import SettingLoader
+from kalliope.core.Models import Singleton
+from kalliope.core.RestAPI.FlaskAPI import FlaskAPI
+
+
+class RestAPITestBase(LiveServerTestCase):
+
+    def tearDown(self):
+        Singleton._instances = {}
+        # clean the lifo
+        LifoManager.clean_saved_lifo()
+
+    def create_app(self):
+        """
+        executed once at the beginning of the test
+        """
+        # be sure that the singleton haven't been loaded before
+        Singleton._instances = {}
```

Comment:
we have the `# be sure that the singleton haven't been loaded before
        Singleton._instances = {}` in the tear down. Why do you need to run it again here ? 
Maybe what you need is a `setUp`

Reply:
All tests are falling if we place this part of code on the setup. Because the singleton is loaded in the "create_app" method and contains all settings of kalliope including the rest api configuration.
But it need to be cleaned here in case other tests ran randomly by python before this one have already configured something.